frontendExternalSystem/src/main/java/com/virtuslab/gitmachete/frontend/externalsystem/project/MacheteProjectResolver.java

```diff
@@ -33,4 +54,16 @@
   public boolean cancelTask(ExternalSystemTaskId taskId, ExternalSystemTaskNotificationListener listener) {
     return false;
   }
+
+  @SuppressWarnings({"guieffect:call.invalid.ui", "interning:not.interned"})
+  private Boolean hasSelectedGitMacheteTab(Project project) {
+    var toolWindowManager = ToolWindowManager.getInstance(project);
+    var toolWindow = toolWindowManager.getToolWindow(ToolWindowId.VCS);
+    if (toolWindow != null) {
+      var contentManager = toolWindow.getContentManager();
```

Comment:
Actually... why suppressing `guieffect:call.invalid.ui` was needed here? Possibly the offending method should have the Checker signature changed in stub files instead... since we likely **treat it as UI-thread-only** while in fact **it's any-thread-safe** (?)